projects/duplicate-char-counter/_posts/2018-12-15-python.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Fizz Buzz in Python
+layout: default
+last-modified: 2020-05-02
+featured-image:
+tags: [python, fizz-buzz]
+authors:
+  - samdoj
```

Comment:
```suggestion
  - GoodbyeBlues
```
Make sure you swap this out with your username. You can define that in the authors.yml file: https://github.com/TheRenegadeCoder/sample-programs-website/blob/master/_data/authors.yml